deltacat/tests/compute/converter/test_convert_session.py

```diff
@@ -639,3 +645,195 @@ def test_converter_pos_delete_multiple_identifier_fields_success(
 
     # Assert elements are same disregard ordering in list
     assert sorted(pk_combined_res) == sorted(expected_result_tuple_list)
+
+
```

Comment:
Can we add a test case to ensure that the sort is stable when multiple files share the same sequence number?

Reply:
Good catch! Will add a secondary sort key based off file path, I noticed the files Spark write out have some sort of prefix in the file name that represents the order of this file within same snapshot, e.g: s3://xxxx/partitionKey=8/primaryKey_bucket=83/**00000-58**-3229fae0-4316-4ea4-9e09-367a5d1b96f9-00001.parquet, the bolded part is observed to be the order of files added within this transaction.